app.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const app = express();
+const {getTopics}= require('./controllers/topics.controllers.js')
+const { handleCustomErrors, handlePsqlErrors, handleServerErrors } = require('./err.js');
+
+app.use(express.json());
```

Comment:
Please remove the express.json(), you are not using it yet as we're not currently dealing with any incoming data.

Reply:
deleted line 6